### ISSUE TITLE
Release 0.8.2

### ISSUE THREAD

**Issue by DanHickstein** [PyAbel/PyAbel#251]
We've incorporated some nice new features and bug-fixes since 0.8.1 was released in Nov 2018. I intend to release 0.8.2 after #248, #248, and #250 have been merged/addressed. If there are more changes that we should make before releasing a new version, please let me know (or open a new issue and reference this one).

**Comment by MikhailRyazanov** [comment]
https://pyabel.readthedocs.io still shows “Copyright 2016” in the footer.

**Comment by DanHickstein** [comment]
I think that copyright date must be the first date that the docs are built on readthedocs. I guess some of the material would be copyright 2016...

Still, this is a good reminder that we need to release v0.8.2! I'll get one that...

**Comment by MikhailRyazanov** [comment]
Technically, the copyright notice is not required at all. On the other hand, there are no other visible dates, so specifying “2016–2019” would indicate that there were major changes during these years, which is more informative that showing only the initial date.

**Comment by DanHickstein** [comment]
I completely agree. But, I don't know how to change this. I think that it's just generated by readthedocs, right?

**Comment by MikhailRyazanov** [comment]
Here:
https://github.com/PyAbel/PyAbel/blob/09479e9ee07d2e6fb6053e7c1ed0d86c4171f547/doc/conf.py#L83

**Comment by DanHickstein** [comment]
Ah, good call. Let's update to "2016-2019"

**Comment by MikhailRyazanov** [comment]
By the way, I have updated the years in #257 (https://github.com/PyAbel/PyAbel/pull/257/commits/c9c4e610c546e6c3fa94534e7d3ede1308bf48fc), so after it is merged, this can be closed.

**Comment by MikhailRyazanov** [comment]
@DanHickstein, since 0.8.2 has been actually released a couple weeks ago, may I suggest to re-purpose this to release 0.8.3?

#257 has added some new capabilities, and #260 has solved a significant usability issue #259, so maybe we should make a new release? Such that people can update PyAbel normally and not suffer if they've updated NumPy recently. (And we also need to add the [RSI reference](https://aip.scitation.org/doi/10.1063/1.5092635) and close #250.)

**Comment by DanHickstein** [comment]
Good idea to make a new release! I'm closing this now that PR #262 exists.
